tests/framework/framework.go

```diff
@@ -139,6 +148,13 @@ func (f *Framework) BeforeEach() {
 		f.CdiClient = cs
 	}
 
+	if f.ControllerPod == nil {
+		pod, err := utils.FindPodByPrefix(f.K8sClient, f.CdiInstallNs, CdiPodPrefix, "")
```

Comment:
In `FindPodByPrefix` is there a way to not examine every pod in the ns in order to see if the cdi controller pod is running? I see that `labelselector` is passed but doesn't seem to be used.

Reply:
This may need to be addressed in a separate pr but there's a minor bug in `FindPodByPrefix` (and you know me, I'd refactor slightly...):
```
...
	err := wait.PollImmediate(2*time.Second, PodCreateTime, func() (bool, error) {
		podList, err := clientSet.CoreV1().Pods(namespace).List(metav1.ListOptions{})
		if err != nil {
                      return false, nil
                }
		var foundPod bool
		for _, pod := range podList.Items {
			if strings.HasPrefix(pod.Name, prefix) {
				if !foundPod {
					foundPod = true
					result = &pod
				} else {
					multiplePods = true
				}
		}
		if foundPod {
			return true, nil
		}
		return false, nil // can't rtn err to Poll because it stops on errors //fmt.Errorf("Unable to find pod starting with prefix %s", prefix)
	})
...
```

Reply:
Yeah I started adding the labelSelector in another PR, but it didn't work or I never finished it. I should fix in separate PR.